### PR TITLE
Check is buf valid

### PR DESCRIPTION
**Pull request by lvim-tech**
Check is buf valid - prevent error

### CODE REVIEW

**Comment by SmiteshP** [comment]
LGTM, but could you share what error occurred and how it occurred?

**Comment by lvim-tech** [comment]
When switch between session:
Screenshot from 2022-07-06 06-46-43

**Review by SmiteshP — APPROVED** [review]
LGTM

File: lua/nvim-navic/init.lua [hunk 1]
```diff
@@ -16,8 +16,10 @@ local function request_symbol(for_buf, handler, client_id)
 					request_symbol(for_buf, handler, client_id)
 				end, 750)
 			elseif symbols[client_id].result ~= nil then
-				vim.b[for_buf].navic_awaiting_lsp_response = false
-				handler(for_buf, symbols[client_id].result)
+				if vim.api.nvim_buf_is_valid(for_buf) then
```
Comment (SmiteshP) [inline lua/nvim-navic/init.lua:19]: would nvim_buf_is_loaded be more appropriate here?

Comment (lvim-tech) [reply lua/nvim-navic/init.lua:19]: I don't understand you

Comment (SmiteshP) [reply lua/nvim-navic/init.lua:19]: Its fine, I will add it my self. Thanks for the PR! 😄

**Review by SmiteshP — APPROVED** [review]

**Pull request merged**